app/notifiers/redis_client.py

```diff
@@ -0,0 +1,52 @@
+"""Notify Alerts via Redis-Stream
+"""
+
+import json
+
+import structlog
+from walrus import Database
+from notifiers.utils import NotifierUtils
+
+
+class RedisNotifier(NotifierUtils):
+    """Used to notify alerts of signals via Redis.
+    """
+
+    def __init__(self, redis_server, redis_port, redis_db, stream):
+        """Initialize redisNotifier class
+
+        Args:
+            redis_server (str): Redis erver Hostname / IP.
+            redis_port   (int): Redis server redis_port
+            redis_db     (int): Redis server database number
+            stream (str): Redis stream name.
+        """
+        if redis_port is None:
+            redis_port = "6379"
+        if redis_db is None:
+            redis_db = "0"
+
+        self.logger = structlog.get_logger()
+        self.connection = Database((host='{}', port='{}', db='{}').format(redis_server, redis_port, redis_db))
```

Comment:
No `.format` needed here I think

```suggestion
        self.connection = Database(host=redis_server, port=redis_port, db=redis_db)
```